helpers/wrapped-key/wrapped_key.py

```diff
@@ -139,9 +137,29 @@ def crc32c(data):
                         'locations/LOCATION-ID'
                         '/keyRings/KEY-RING-ID/cryptoKeys/KEY-ID')
 
+    group3.add_argument('--service_account', dest='service_account',
+                        help='service_account (string): '
+                        'Service Account to be impersonated.')
+
     args = parser.parse_args()
-    if args.crypto_key_path is not None:
+
+    # Create the client.
+    if args.service_account is not None:
+        target_scopes = ['https://www.googleapis.com/auth/cloud-platform']
+
+        source_credentials, project = google.auth.default(
+            scopes=target_scopes)
+
+        target_credentials = impersonated_credentials.Credentials(
+            source_credentials=source_credentials,
+            target_principal=args.service_account,
+            target_scopes=target_scopes,
+            lifetime=500)
```

Comment:
We check and the python script takes 5 seconds to run. 

To guarantee, we set the impersonate lifetime to 10 seconds, in case the python script takes more time to run.